### ISSUE TITLE
Multiple API specs loaded from URLs at separate routes do not work

### ISSUE THREAD

**Issue by richiksc** [scottie1984/swagger-ui-express#305]
If I try to have two routes that serve from SwaggerUI express that each load from separate OpenAPI spec URLs, the same API spec and spec URL is shown for both routes:

```js
const swaggerUiOptsA = {
    swaggerOptions: { url: '/path/to/specA' }
}

const swaggerUiOptsB = {
    swaggerOptions: { url: '/path/to/specB' }
}

router.use('/', swaggerUi.serveFiles(null, swaggerUiOptsA));
router.get('/', swaggerUi.setup(null, swaggerUiOptsA));

router.use('/service-b', swaggerUi.serveFiles(null, swaggerUiOptsB));
router.get('/service-b', swaggerUi.setup(null, swaggerUiOptsB));
```

Then, accessing both `/` and `/service-b` displays the docs rendered from `/path/to/specA` and display `/path/to/specA` as the OpenAPI spec URL on the page.

This behavior works if I pass in an actual JSON object as the swaggerDocument parameter for each of them, but for my use case, I need the specs loaded from the URLs.

**Comment by danielnmai** [comment]
I ran into  this issue as well, swagger UI always show the last spec file, even there are 2 different files and they're in different routes. There is a workaround I came up with: using the explorer dropdown to select the spec file you want. It still has 1 single route, so not sure it would fit your solution. Here's the code:

`  app.get('/api-docs/specA.json', (req, res) => res.json(specA));
app.get('/api-docs/specB.json', (req, res) => res.json(specB));

  const options = {
    explorer: true,
    swaggerOptions: {
      urls: [
        {
          url: '/api-docs/specA.json',
          name: 'Spec A',
        },
        {
          url: '/api-docs/specB.json',
          name: 'Spec B',
        },
      ],
    },
  };

router.use('', swaggerUI.serve, basicAuth.check(swaggerUI.setup(undefined, options)));

  app.use('/v1/api-docs', router);

`
